content/api/entity-reference/volunteer/index.html

```diff
@@ -3,7 +3,7 @@
 name: Volunteer Entity Reference
 title: Volunteer Entity Reference
 order: 90
-published: false
+published: true
```

Comment:
whoops good catch, i had to turn it on to be able to see it and forgot to turn it off